### ISSUE TITLE
Emulated captions with <audio> elements display tiny subtitles 

### ISSUE THREAD

**Issue by hartman** [videojs/video.js#3390]
While looking further into #2955 I noticed that for me on Safari, I actually do get emulated captions displaying for <audio> elements. They are simply uselessly small, since they are relative to the player dimensions, and the player dimensions for audio players are usually already very small.

I think we should probably not pass the _el to processCues, when we are an <audio> element (there is no display area). This brings as closer to how browsers behave.

It would also be nice to have a workaround for #2955, but that should probably not be the default, but an extra option ?

**Comment by gkatsev** [comment]
Yeah, we should probably just not display anything when we're in audio mode.

**Comment by OwenEdwards** [comment]
It's `vtt.js` which decides what percentage of the player window will be used by captions (I think it's 5%). This makes sense for video, but for audio, if there *is* a poster where the captions can be displayed, then this probably ought to be 80% or 100%. Right now there's no API to vtt.js to allow this change.
